src/components/Modal/ModalBasic/ModalBasic.tsx

```diff
@@ -0,0 +1,40 @@
+import ModalHeader from "./ModalHeader";
+import ModalMain from "./ModalMain";
+import styles from "./ModalBasic.module.scss";
+import CloseIcon from "@/assets/svg/CloseIcon";
+import { RefObject } from "react";
+
+interface ModalBasic {
+  name: string;
+  modalRef: RefObject<HTMLDivElement>;
+  handleClickOutside: () => void;
+
+  subtitle?: string;
+  shareButtons?: boolean;
+}
+
+export default function ModalBasic({
+  name,
+  modalRef,
+  handleClickOutside,
+  ...rest
+}: ModalBasic) {
+  const { subtitle, shareButtons } = rest;
+  return (
+    <>
+      <div
+        className={styles.modalBackground}
+        onClick={handleClickOutside}
+      ></div>
```

Comment:
html에 컨텐츠가 없으면 이렇게 self-closing으로 표현해주는게 좋습니다~
```suggestion
      <div
        className={styles.modalBackground}
        onClick={handleClickOutside}
      />
```